source/utilities.tex

```diff
@@ -16304,7 +16304,7 @@
             int, unsigned int, long long int, unsigned long long int,
             float, double, long double,
             const @\exposid{char-type}@*, basic_string_view<@\exposid{char-type}@>,
-            const void*, handle> value;                                         // \expos
+            const void*, handle> @\exposid{value_}@;                                         // \expos
```

Comment:
```suggestion
            const void*, handle> @\exposid{value_}@;                                        // \expos
```

Reply:
Sorry, I don't see the difference.  Is this a parallel comment with the commit I just pushed?

Reply:
No. You added a `_`, and I suggested removing a space before the `// expos` to keep it aligned.